docker/lnd/start-lnd.sh

```diff
@@ -42,16 +42,17 @@ set_default() {
 RPCUSER=$(set_default "$RPCUSER" "devuser")
 RPCPASS=$(set_default "$RPCPASS" "devpass")
 DEBUG=$(set_default "$DEBUG" "debug")
-BITCOIN_NETWORK=$(set_default "$BITCOIN_NETWORK" "simnet")
+NETWORK=$(set_default "$NETWORK" "simnet")
+CHAIN=$(set_default "$CHAIN" "bitcoin")
 
 lnd \
     --datadir="/data" \
     --logdir="/data" \
-    --bitcoin.active \
-    "--bitcoin.$BITCOIN_NETWORK" \
-    --bitcoin.rpchost="btcd" \
-    --bitcoin.rpccert="/rpc/rpc.cert" \
-    --bitcoin.rpcuser="$RPCUSER" \
-    --bitcoin.rpcpass="$RPCPASS" \
+    "--$CHAIN.rpccert"="/rpc/rpc.cert" \
+    "--$CHAIN.active" \
+    "--$CHAIN.$NETWORK" \
+    "--$CHAIN.rpchost"="blockchain" \
```

Comment:
Tested this locally, and looks like it does indeed to to be scope somehow. 

Run from inside the container:
```
root@aec17b768d6e:/go/src/github.com/lightningnetwork/lnd# ps xa
  PID TTY      STAT   TIME COMMAND
    1 ?        Ss     0:00 bash ./start-lnd.sh
   11 ?        Sl     0:04 lnd --datadir=/data --logdir=/data --bitcoin.rpccert=/rpc/rpc.cert --bitcoin.active --bitcoin.simnet --bitcoin.rpchost=blockchain --bitcoin.rpcuser=devuser --bitcoin.rpcpass=devpass --d
   27 pts/0    Ss     0:00 bash
   33 pts/0    R+     0:00 ps xa
```
The point of interest is: `--bitcoin.rpchost=blockchain`, which means the string was passed through without any sort of substitution.